### DISCUSSION TITLE
Priority Items

### DISCUSSION THREAD

**Discussion by wjwat** [wjwat/lhtlhtp.com#5] [Ideas]
- Documentation to bring people up to speed on how to help/contribute
  - Using Zola
  - How to submit an issue/pull request
  - Style & standards guide for content
    - keep the tone light
    - not too technical or dense
    - etc etc
- Styling needs to be updated for:
  - tables
  - footnotes
  - quote blocks(?)
  - Add a shortcode for contributors

**Comment by wjwat** [comment]
e424292

Made some progress on tackling the contributing section today. Also cleaned up the site structure a bit. Feedback always appreciated!

**Comment by wjwat** [comment]
Filled in a bit more of contributing, and started going... somewhere in the intro section, but may need help to clarify thoughts.

**Reply by probablynotryan** [reply]
The latest push did a great job clarifying things and inviting people to contribute, to be honest.
